CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ catkin_package(
 ## Build ##
 ###########
 
-include_directories(${THIS_PACKAGE_INCLUDE_DIRS} ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIR} ${EIGEN3_INCLUDE_DIRS})
+include_directories(${THIS_PACKAGE_INCLUDE_DIRS} ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIR} ${EIGEN3_INCLUDE_DIRS} ${EmptyPlan_INCLUDE_DIRS} )
```

Comment:
im pretty sure you don't need this include, as catkin_INCLUDE_DIRS does this for you right? @mlautman